build/Common.nonprod.props

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DefaultTargetFrameworkForExampleApps>net8.0</DefaultTargetFrameworkForExampleApps>
+    <DefaultTargetFrameworkForExampleApps>net9.0</DefaultTargetFrameworkForExampleApps>
```

Comment:
is this necessary?

Reply:
Yes, it is used in all example projects.

```
<TargetFramework>$(DefaultTargetFrameworkForExampleApps)</TargetFramework>
```